### PR TITLE
Multisite deployment with multipe rgws via HAproxy.

### PR DESCRIPTION
**Pull request by viduship**
# Description
- This PR addresses **T2 automation-CEPH-83574575-RGW MS with HAProxy endpoints.**

- This would also address using multiple rgws listening behind a LoadBalancer.
   and issue : https://github.com/red-hat-storage/ceph-qe-scripts/issues/191

- It would configure multisite with haproxy endpoint and run the IOs via the haproxy node.

- We can use "**run-on-haproxy**" in the config file to run IOs via Haproxy node.

Pass logs: 
1. http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LA38TE/ ( the failed test was because rgws were not restarted after haproxy.), hence have included the test in the end (in the suite file as well.)
2. http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NKG95F
3. log without HAproxy for script bucket_listing.py --> http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PJL77R

### CODE REVIEW

File: pipeline/metadata/6.1.yaml [hunk 1]
```diff
@@ -726,6 +726,17 @@ pipelines:
...
+          execution_time: "2h 29m 53s"
+          suite: "suites/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy.yaml"
+          global-conf: "conf/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy_conf.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rgw
+
```
Comment (anrao19) [inline pipeline/metadata/6.1.yaml:739]: Please update the metadata of 5.3 and 6.0, since suite is in pacific and part of 6.2 metadata file

**Comment by ckulal** [comment]
With the changes made in this PR, request to provide  log for any other suite..other than test_Mbuckets_with_Nobjects.py,
where Ha-proxy changes are not yet introduced

File: pipeline/metadata/6.1.yaml (hunk 1 above)
Comment (viduship) [reply pipeline/metadata/6.1.yaml:739]: @anrao19 Done.

**Review by anrao19 — APPROVED** [review]

**Review by ckulal — COMMENTED** [review]
looks good

**Review by hmaheswa — APPROVED** [review]
Looks Good

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**